crates/sui-core/src/consensus_handler.rs

```diff
@@ -190,6 +205,8 @@ impl<C> ConsensusHandler<C> {
 impl<C: CheckpointServiceNotify + Send + Sync> ConsensusHandler<C> {
     #[instrument(level = "debug", skip_all)]
     async fn handle_consensus_commit(&mut self, consensus_commit: impl ConsensusCommitAPI) {
+        self.backpressure_subscriber.await_backpressure().await;
```

Comment:
Comment that this can block until pending insertions in writeback cache is less than a threshold?